pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -193,199 +221,87 @@ def get_commuter_data(read_data=dd.defaultDict['read_data'],
+            checksum = 0  # sum of county migration from, to be checked against sum in document
+
+            for i in range(0, commuter_migration_file.shape[0]):
+                if (len(str(commuter_migration_file.iloc[i][0])) == 5
+                        and (commuter_migration_file.iloc[i][0]).isdigit()):
+                    checksum = 0
+                    # make zero'd list of counties explicitly migrated to from county considered
+                    # 'implicit' migration means 'migration to' which is summed in a larger regional entity and not given in
+                    # detail per county
+                    counties_migratedfrom = []
+                    for j in range(0, len(gov_county_table)):
+                        counties_migratedfrom.append(
+                            np.zeros(len(gov_county_table[j])))
+
+                    counties_done.append(commuter_migration_file.iloc[i][0])
+                    current_col = countykey2numlist[commuter_migration_file.iloc[i][0]]
+                    curr_county_migratedto = commuter_migration_file.iloc[i][1]
+                    current_key = commuter_migration_file.iloc[i][0]
+                    # migration to itself excluded!
+                    counties_migratedfrom[countykey2govkey[current_key]][
+                        countykey2localnumlist[current_key]] = 1
+
+                if not isinstance(commuter_migration_file.iloc[i][2], float):
+                    # removal of nan's, regional keys are stored as strings
+
+                    # check if entry is a digit
+                    if (commuter_migration_file.iloc[i][2]).isdigit():
+                        # explicit migration from county to county
+                        if len(str(commuter_migration_file.iloc[i][2])) == 5:
+                            # check if entry refers to a specific county, then set matrix value
+                            current_row = countykey2numlist[commuter_migration_file.iloc[i][2]]
+                            # TODO
+                            val = commuter_migration_file.iloc[i][4]
+                            mat_commuter_migration[current_row,
+                                                   current_col] = val
+                            checksum += val
+                            counties_migratedfrom[countykey2govkey[commuter_migration_file.iloc[i][2]]
+                                                  ][countykey2localnumlist[commuter_migration_file.iloc[i][2]]] = 1
+
+                        # take summed values of other REMAINING counties of government region
+                        # here, some counties of the region are stated explicitly and the rest is summed
+                        elif (str(commuter_migration_file.iloc[i][3]) == 'Übrige Kreise (Regierungsbezirk)' and str(
+                                commuter_migration_file.iloc[i][4]).isdigit()):
+                            # remove trailing zeros (dummy key w/o zeros: dummy_key_wozeros)
```

Comment:
```suggestion
                            # remove trailing zeros (dummy key w/o zeros: dummy_key_non_zero)
```
I think the name is not optimal :).

Reply:
I agree that the naming is not good here but dummy_key_non_zero is also missleading. If the key is `010` then `dummy_key_wozeros = 01`.